.eslintrc-es6

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "es6": "off"
```

Comment:
why is a file named "es6" turning off the es6 env?

also, why is this a separate file at all, instead of just being in the root eslintrc?